tests/playbooks/tests_routing_rules.yml

```diff
@@ -28,89 +28,134 @@
+                    - priority: 30202
+                      family: ipv4
+                      ipproto: 6
+                      table: 30200
+                    - priority: 30203
+                      family: ipv4
+                      sport: 128 - 256
+                      table: 30200
+                    - priority: 30204
+                      family: ipv4
+                      tos: 8
+                      table: 30200
+                    - priority: 30205
+                      uid: 2000 - 3000
+                      family: ipv4
+                      table: 30200
+                    - priority: 30206
+                      suppress_prefixlength: 8
+                      family: ipv4
+                      table: 30200
+                    - priority: 30400
+                      to: 198.51.100.128/26
+                      table: 30400
+                    - priority: 30401
+                      family: ipv4
+                      iif: iiftest
+                      table: 30400
+                    - priority: 30402
+                      family: ipv4
+                      oif: oiftest
+                      table: 30400
+                    - priority: 30403
+                      from: 0.0.0.0/0
+                      to: 0.0.0.0/0
+                      table: 30400
+                    - priority: 30600
+                      to: 2001:db8::4/32
+                      table: 30600
+                    - priority: 30601
+                      family: ipv6
+                      dport: 128 - 256
+                      invert: true
+                      table: 30600
+                    - priority: 30602
+                      from: ::/0
+                      to: ::/0
+                      table: 30600
+                    - priority: 30603
+                      uid: 6000 - 8000
+                      family: ipv6
+                      table: 30600
+                    - priority: 30604
+                      suppress_prefixlength: 24
+                      family: ipv6
+                      table: 30600
+                    - priority: 200
+                      from: 198.51.100.56/26
+                      table: custom
+
+      rescue:
+        - name: Assert that the routing rule attribute 'suppress_prefixlength'
+              validation failure is not raised when the distro's major version is 7
+          assert:
+            that:
+              - __network_connections_result.stderr is search("the routing rule
+                selector 'suppress_prefixlength' is not supported in NetworkManger
+                until NM 1.20")
+            msg: The routing rule attribute 'suppress_prefixlength' validation
+              failure is not raised when the distro's major version is 7
+          when: ansible_distribution_major_version == "7"
+
+        - name: Clear errors
+          meta: clear_host_errors
+
+        - name: Reset __network_end_play variable
+          set_fact:
+            __network_end_play: true
+
```

Comment:
need to delete the routing table file here or it won't be removed (at least, if it is, I can't tell where it is removed)  path: /etc/iproute2/rt_tables.d/table.conf

Reply:
I reorganized the route table cleanup task, so that it is guaranteed to run.